### ISSUE TITLE
New set of boronic acids based on new hit OSM-S-525 + some others

### ISSUE THREAD

**Issue by maratsydney** [OpenSourceMalaria/Series4#31]
Bio activity results from the last batch of  compounds #26 revealed new potent molecule OSM-525 (IC50= 86nM) that posses indole fragment on the right hand side. Looking through available boronic acids I found that structures **1-6** can be easily made at reasonable price.  I would like to hear any suggestions on what else can be made in this direction.
As well compound **7** will be made for further late state functionalization and **8-9** to check impact of carboxy group on the activity. 

![2018 03 26 boronic acid to buy](https://user-images.githubusercontent.com/29498087/37904618-1a7c5222-3148-11e8-82ea-1911001908aa.png)
----
FC1=CC(CCOC2=CN=CC3=NN=C(C4=CC=C5C(NC=C5)=C4)N32)=CC=C1F
InChI=1S/C21H15F2N5O/c22-16-4-1-13(9-17(16)23)6-8-29-20-12-24-11-19-26-27-21(28(19)20)15-3-2-14-5-7-25-18(14)10-15/h1-5,7,9-12,25H,6,8H2
VHFREJHLTSVZGA-UHFFFAOYSA-N

**Comment by MFernflower** [comment]
If you still have some OSM-525 on hand could try methylating free N?

**Comment by drc007** [comment]
@maratsydney It might be worth considering the 5-azaindole analogue. The pKa of the pyridine nitrogen is about 8.3 and so weakly basic, if active it could aid solubility.

**Comment by MFernflower** [comment]
benzoxadiazole might be interesting to try and does have precedent in drug design: https://en.wikipedia.org/wiki/Farampator

**Comment by mcoster** [comment]
Interesting suggestion @MFernflower. What would the oxindole look like using that software?

![osm - oxindole](https://user-images.githubusercontent.com/28282596/37942356-00ffe458-31b6-11e8-9958-651cb1b52a15.png)

**Comment by MFernflower** [comment]
Curently in bed software is www.swissadme.ch also might be wise to remove
those two LHS f atoms because make stuff oily @mcostner

**Comment by MFernflower** [comment]
I wonder if indole so potent because possible interaction with a PHE on the
pump? @holeng if you have free time could you dock osm-s-525?

**Comment by mcoster** [comment]
OK, I finally dived in and played with SwissADME, but then I got distracted by the other Swiss apps... 
 (not the Swiss Alps!..) [SwissBioisostere](http://www.swissbioisostere.ch/) is an interesting tool for analogue planning. I was interested to see an example of an indole -> N-methyloxindole replacement (different regio-isomer) lead to a more active compound with lower logP. This looks like a handy tool for those at the synthesis coal-face of these analogues. @maratsydney 

![image](https://user-images.githubusercontent.com/28282596/37945144-16c596d2-31c2-11e8-858f-97f816a03415.png)

**Comment by MFernflower** [comment]
It's a great software suite innit?

**Comment by MFernflower** [comment]
@drc007 does position of the nitrogen atom affect the desired proprieties of the ring system - if not this compound is really easy to get http://www.matrixscientific.com/boronic-acids/068655.html

**Comment by maratsydney** [comment]
Thank you for your ideas!
I summarize all the suggestion in chemdraw file shown below. 
 @MFernflower Yes, I can try to methylate OSM-S-525.  Benzoxadiazole seems to be a big change, but probably worth to try. 
@drc007 Unfortunately I didn't find 5-azaindole, but there are some on the market with nitrogen in other position (**11-13**) and since indole nitrogen will be in different position I guess it's will be reasonable to test compound 2 and see the activity.
@mcoster oxindole (**10**) definitely looking good to me.
Tried to play with SwissBioisostere and it suggested N->O substitution from indole to bezofuran. This change only interesting in order to check tolerance to other electron donor atom.

Since methodology to make it from boronic acids is reliable and simple the only concern is cost of precursor, so I would like to hear opinion on the second set of compounds what would be more interesting to test  in the first place?
![2018 03 28boronic acids](https://user-images.githubusercontent.com/29498087/38009839-13fb68a6-32a2-11e8-95ea-a4ea4bba1366.png)

[boronic acids.zip](https://github.com/OpenSourceMalaria/Series4/files/1854638/boronic.acids.zip)

**Comment by mattodd** [comment]
I like them. Great job. I also like 3, 6, 8 and 9 from the first scheme.

Some of the above red fragments are, I think, available because they are widely used in fragment-based drug discovery screens.

**Comment by MFernflower** [comment]
"Some of the above red fragments are, I think, available because they are widely used in fragment-based drug discovery screens."

You would be correct as massive fragment library love to use easy reactions like Suzuki @mattodd

**Comment by MFernflower** [comment]
What I am about to post is rather pedantic but I still feel important to point out 
![38009839-13fb68a6-32a2-11e8-95ea-a4ea4bba1366](https://user-images.githubusercontent.com/3164942/38066805-a419bcc8-32d7-11e8-8c1b-7a23e8442f0d.png)

**Comment by cdsouthan** [comment]
Sorry to sound curmudgeonly but can we a) remake some OSM-S-525 b) retest both the old batch and the new batch in //,  just saying......

**Comment by mattodd** [comment]
See #26 for some suggestions relevant to the above that just came in by email from Neil Norcross.

**Comment by MFernflower** [comment]
My votes are for:

Mark and Neil's oxindole 
My benzofurazan compound 
The **benzofuran** compound that the computer program suggested
and the N-methyl indole

**Comment by drc007** [comment]
A few thoughts.

Anilines have a deserved reputation for genotoxicity.

Does the inactivity of the 3-acetanilide MMV1579346 (>25 uM) suggest that the indole NH of MMV1579341 (0.086 uM) is not acting as critical H-bond donor? Might be useful to retest both compounds again to confirm activities. Also agree simple methylation of indole should be target.

I always regard indole as an electron-rich heterocycle, perhaps increased affinity is due to pi-stacking? perhaps replace with simple 3-anisole?

Any plans to introduce hydroxylated side-chain of MMV672687 (0.074 uM)?

It might also be worth retesting the unsubstituted compound MMV1557949 (2.15 uM) to make sure we have a solid baseline number.

**Comment by MFernflower** [comment]
I agree 200% @drc007 I think we could get away with swapping indole N with C or O as if these are potent it will confirm we only need a aromatic heterocycle that only needs to be roughly the size and shape of indole

Also I really think it would be wise to move away from free RHS benzeneamines with no N methyl groups - free arylamines are known mutagens (eg napthylamine)

**Comment by MFernflower** [comment]
I know that the compounds I am proposing will be rather greasy but a question that has been nagging me is if we can simply just tack a medium sized non-metabolizable functional group on the RHS benzene ring para position and still retain potency? I know Sf5 worked so why not attempt t-bu and tms? I know the precursors are pricey but I want to put this out there so people can look at the ideas from a less starry eyed perspective on C-Si chemistry???? Have I been watching too much X-files or is this something you think could be pursued for yet more SAR expansion?
